docs/features/techdocs/using-cloud-storage.md

```diff
@@ -195,3 +195,92 @@ Your Backstage app is now ready to use AWS S3 for TechDocs, to store and read
 the static generated documentation files. When you start the backend of the app,
 you should be able to see
 `techdocs info Successfully connected to the AWS S3 bucket` in the logs.
+
+## Configuring Azure Blob Storage Container with TechDocs
+
+Follow the
+[official Azure Blob Storage documentation](https://docs.microsoft.com/en-us/azure/storage/common/storage-auth?toc=/azure/storage/blobs/toc.json)
+for the latest instructions on the following steps involving Azure Blob Storage.
+
+**1. Set `techdocs.publisher.type` config in your `app-config.yaml`**
+
+Set `techdocs.publisher.type` to `'azureBlobStorage'`.
+
+```yaml
+techdocs:
+  publisher:
+    type: 'azureBlobStorage'
+```
+
+**2. Create an Azure Blob Storage Container**
+
+Create a dedicated container for TechDocs sites.
+[Refer to the official documentation](https://docs.microsoft.com/pt-br/azure/storage/blobs/storage-quickstart-blobs-portal).
+
+TechDocs will publish documentation to this container and will fetch files from
+here to serve documentation in Backstage. Note that the container names are
+globally unique.
+
+Set the config `techdocs.publisher.azureBlobStorage.containerName` in your
+`app-config.yaml` to the name of the container you just created.
+
+```yaml
+techdocs:
+  publisher:
+    type: 'azureBlobStorage'
+    azureBlobStorage:
+      containerName: 'name-of-techdocs-storage-container'
+```
+
+**3a. (Recommended) Authentication using environment variable**
+
+Set the config `techdocs.publisher.azureBlobStorage.credentials.accountName` in
+your `app-config.yaml` to the your account name.
+
```

Comment:
Let's also add this link https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-create-service-principal-portal here since it has the steps to create the service where the `AZURE_TENANT_ID`, `AZURE_CLIENT_ID` and `AZURE_CLIENT_SECRET` can be retrieved from.